### PR TITLE
Remove unused datasets

### PR DESCRIPTION
**Pull request by utkarsharma2**
# Description
## What is the current behavior?
We are not using this dataset for benchmarking, we should remove it. Also, there was a failure for the StackExchange dataset.

### CODE REVIEW

File: python-sdk/tests/benchmark/datasets.md [hunk 1]
```diff
@@ -140,95 +140,3 @@ The [script](synthetic_ndjson_generator.py) is used to generate the NDJSON file.
 | five_gb    | 4.5 GB | NDJSON   | s3://astro-sdk/benchmark/synthetic-dataset/ndjson/five_gb.ndjson    |
 | five_gb    | 5 GB   | NDJSON   | s3://astro-sdk/benchmark/synthetic-dataset/ndjson/five_gb/          |
 | ten_gb     | 10 GB  | NDJSON   | s3://astro-sdk/benchmark/synthetic-dataset/ndjson/ten_gb/           |
-
-## Origin of the real datasets
```
Comment (sunank200) [inline python-sdk/tests/benchmark/datasets.md:144]: This dataset was originally used as a realistic dataset. Shouldn't we still keep it?

**Review by sunank200 — COMMENTED** [review]
This dataset was originally used as a realistic dataset. Shouldn't we still keep it?

**Comment by tatiana** [comment]
End-users use real datasets, which has allowed us to surface several issues with our stack. Much more than any synthetic dataset we have.

I believe we should have end-to-end tests that use these datasets instead of simply getting rid of them.

**Comment by utkarsharma2** [comment]
I agree @tatiana , but currently, we are not using it in our code base anywhere. When do we start doing that we can add them again to the codebase, WDYT? we can maybe create a ticket(s) to add a test case with a real dataset?

File: python-sdk/tests/benchmark/datasets.md (hunk 1 above)
Comment (utkarsharma2) [reply python-sdk/tests/benchmark/datasets.md:144]: We are not adding those numbers obtained by these datasets in benchmarking results either. right?

**Comment by tatiana** [comment]
I'm happy for them to be removed and re-introduced as long as we have a task logged for us to have e2e tests using realistic datasets, referencing this PR :)

**Comment by utkarsharma2** [comment]
@tatiana Added a ticket for the same - https://github.com/astronomer/astro-sdk/issues/1612

**Review by tatiana — APPROVED** [review]

**Review by pankajastro — COMMENTED** [review]
Most of these data we have stored in gcs do you want to keep it or we should clean them also @utkarsharma2 ?

**Comment by utkarsharma2** [comment]
> Most of these data we have stored in gcs do you want to keep it or we should clean them also example gs://astro-sdk/benchmark/trimmed/github/github_timeline_100mb.csv @utkarsharma2 ?

We should do that as well. I'll merge this PR for now. I'll create a separate ticket for this.

**Review by pankajkoti — APPROVED** [review]

**Review by pankajastro — APPROVED** [review]

**Pull request merged**